graphing/html/index.html

```diff
@@ -3,60 +3,135 @@
+            // });
             const myChart = new Chart(ctx, {
-                type: 'bar',
+                type: 'line',
                 data: {
-                    labels: ['Red', 'Blue', 'Yellow', 'Green', 'Purple', 'Orange'],
+                    labels: ['1', '2', '3', '4', '5', '6', '7','8', '9', '10', '11','12'],
                     datasets: [{
-                        label: '# of Votes',
-                        data: [12, 19, 3, 5, 2, 3],
+                        label: false,
+                        data: [0.6, 0.75, 0.4, 0.65, 0.9, 1.1, 0.78, 1.5, 1.05, 0.4, 0.8, 0.6],
+                        lineTension: 0,
+                        borderColor: 'rgb(128,23,25)',
+                        fill: false,
                         backgroundColor: [
-                            'rgba(255, 99, 132, 0.2)',
-                            'rgba(54, 162, 235, 0.2)',
-                            'rgba(255, 206, 86, 0.2)',
-                            'rgba(75, 192, 192, 0.2)',
-                            'rgba(153, 102, 255, 0.2)',
-                            'rgba(255, 159, 64, 0.2)'
-                        ],
-                        borderColor: [
-                            'rgba(255, 99, 132, 1)',
-                            'rgba(54, 162, 235, 1)',
-                            'rgba(255, 206, 86, 1)',
-                            'rgba(75, 192, 192, 1)',
-                            'rgba(153, 102, 255, 1)',
-                            'rgba(255, 159, 64, 1)'
+                            //'rgba(255, 99, 132, 0.2)',
+                            // 'rgba(54, 162, 235, 0.2)',
+                            //'rgba(255, 206, 86, 0.2)',
+                             'rgba(75, 192, 192, 0.2)',
+                            // 'rgba(153, 102, 255, 0.2)',
+                            // 'rgba(255, 159, 64, 0.2)'
                         ],
-                        borderWidth: 1
+                        borderWidth: 5
                     }]
                 },
                 options: {
+                    legend: {
+                        display: false,
+                    },
+                    title: {
+                        display: true,
+                        text: 'Median time to first view after first sustained eCART elevation, hrs'
+                    },
+                    lineAt: 1,
+                    elements: {
+                        point:{
+                            radius: 0
+                        }
+                    },
+                    annotation: {
+                        annotations: [{
+                            type: 'box',
+                            drawTime: 'beforeDatasetsDraw',
+                            id: 'region-1',
+                            xScaleID: 'x-axis-0',
+                            yScaleID: 'y-axis-0',
+                            xMin: '1',
+                            xMax: '12',
+                            yMin: 0.5,
+                            yMax: 1,
```

Comment:
These values are also configuration for the graph so they should be sent from the API. Pretty much anything that the user will want to configure should be loaded from the API route.